src/sentry/integrations/gitlab/client.py

```diff
@@ -70,6 +72,7 @@ def request(self, method, path, data=None, params=None):
 
 
 class GitLabApiClient(ApiClient):
+    base_url = "https://gitlab.com"
```

Comment:
@ceorourke isn't this going to not work for anyone who has on-premise Gitlab?

Reply:
oh ... good point, I'm not sure. `get_cached` was failing without this because `build_url` requires a `base_url`. I'll look into how to make that work for on prem users too.

Reply:
@ceorourke You might need to update the implementation of `get_cached` to be able to work with Gitlab.